https://bugs.gentoo.org/580188

From 5c97e465b33bf8fefb17da7f553a1c3812e508d5 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Tue, 19 Apr 2016 18:10:54 -0400
Subject: [PATCH] include sys/sysmacros.h for major/minor/makedev prototypes

These funcs are defined in sys/sysmacros.h, so include it for them.
It works today due to an implicit include by sys/types.h, but the
various Linux C libraries are moving away from that.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 mdadm.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mdadm.h b/mdadm.h
index d209488..b8767a7 100755
--- a/mdadm.h
+++ b/mdadm.h
@@ -33,6 +33,7 @@ extern __off64_t lseek64 __P ((int __fd, __off64_t __offset, int __whence));
 # endif
 #endif
 
+#include	<sys/sysmacros.h>
 #include	<sys/types.h>
 #include	<sys/stat.h>
 #include	<stdint.h>
-- 
2.7.4

